api/src/main/java/com/epam/pipeline/manager/resource/StaticResourcesService.java

```diff
@@ -46,7 +68,54 @@ public DataStorageStreamingContent getContent(final String path) {
         Assert.isTrue(StringUtils.isNotBlank(filePath),
                 messageHelper.getMessage(MessageConstants.ERROR_STATIC_RESOURCES_INVALID_PATH));
         final AbstractDataStorage storage = dataStorageManager.loadByNameOrId(bucketName);
+        if (Files.isDirectory(Paths.get(path))) {
```

Comment:
This check won't work for cloud resource, I will add a new method to check cloud object type